.rancher-pipeline.yml

```diff
@@ -0,0 +1,27 @@
+stages:
+  - name: Build
+    steps:
+      - runScriptConfig:
+          image: maven:3-openjdk-11
+          shellScript: mvn package -DskipTests
+  - name: Build Docker with DIND
+    steps:
+      - publishImageConfig:
+          dockerfilePath: ./Dockerfile
+          buildContext: .
+          tag: docker.dev.folio.org/mod-inventory:spitfire-${CICD_EXECUTION_SEQUENCE}
+          pushRemote: true
+          registry: docker.dev.folio.org
+  - name: Deploy
+    steps:
+      - applyAppConfig:
+          catalogTemplate: p-htqfq:spitfire-helmcharts-mod-inventory
+          version: 0.1.32
+          answers:
+            image.repository: docker.dev.folio.org/mod-inventory
+            image.tag: spitfire-${CICD_EXECUTION_SEQUENCE}
+            postJob.enabled: true
+          targetNamespace: spitfire
+          name: mod-inventory
+timeout: 60
+notification: {}
```

Comment:
Please remove this before the branch is merged into the mainline. Other teams might be using scratch / rancher environments and have their own configuration